### PR TITLE
Improved the readme

### PR DESCRIPTION
**Pull request by Raffo**
Thanks @LappleApple for help!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
-This ingress controller runs inside a Kubernetes cluster monitoring changes to the Ingress resources and orchestrating 
-AWS Load Balancers accordingly.
+kube-ingress-aws-controller uses your EC2 instance metadata to find the CloudFormation stack that it belongs to, and then uses this information to discover the required AWS resources that are attached to each newly created [Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/applicationloadbalancer/).
 
-It will use the EC2 instance metadata to find the CloudFormation
-stack it belongs to which is then used to discover the required AWS resources that are attached to each newly created Application Load Balancer.
+## Development Status
 
-## How it works
+kube-ingress-aws-controller is a work in progress. As of March 2017 it's being used only in small production use cases at [Zalando](https://www.zalando.de/), and is not yet battle-tested. However, we are actively seeking people to try it out and share feedback so we can make it better. We are also eager to bring new contributors on board; see our contributor guidelines here.
```
Comment (hjacobs) [inline README.md]: Link to tech.zalando.com?

File: README.md [hunk 2]
```diff
@@ -48,9 +60,9 @@ spec:
           servicePort: main-port
 ```
 
-The Application Load Balancer that will be created, will have both an HTTP listener and an HTTPS listener. The latter, will use the automatically selected certificate.
+The Application Load Balancer that will be created by the controller, will have both an HTTP listener and an HTTPS listener. The latter, will use the automatically selected certificate.
 
-Alternatively, the ARN of the desired certificate can be specified with an annotation like in the following example: 
+Alternatively, you can specify the Amazon Resource Name of the desired certificate can be specified with an annotation like in the following example:
```
Comment (hjacobs) [inline README.md]: I would still add "(ARN)" after "Amazon Resource Name" as "ARN" is more readable for people familiar with AWS

File: README.md (hunk 1 above)
Comment (LappleApple) [inline README.md]: +1

File: README.md (hunk 2 above)
Comment (LappleApple) [inline README.md]: +1

**Comment by LappleApple** [comment]
@Raffo @hjacobs  can you merge so I can do my next round of edits?

File: README.md (hunk 2 above)
Comment (lmineiro) [inline README.md]: Agree. We don't need to specify "Amazon Resource Name" every time. Besides, any potential user will require a certain level of knowledge about the AWS environment to understand what ARN stands for.

**Comment by hjacobs** [comment]
:+1:

File: README.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ spec:
 
 The Application Load Balancer that will be created by the controller, will have both an HTTP listener and an HTTPS listener. The latter, will use the automatically selected certificate.
 
-Alternatively, you can specify the Amazon Resource Name of the desired certificate can be specified with an annotation like in the following example:
+Alternatively, you can specify the Amazon Resource Name (ARN) of the desired certificate can be specified with an annotation like in the following example:
```
Comment (lmineiro) [inline README.md]: This reads wrong.

"Alternatively, you can specify the Amazon Resource Name (ARN) of the desired certificate with an annotation like in the following example:"

**Comment by hjacobs** [comment]
:+1:

**Comment by lmineiro** [comment]
👍

**Pull request merged**